### PR TITLE
Add django 3.0 to the test matrix and drop six

### PR DESCRIPTION
**Pull request by mvbrn**
## Problem

Six won't be used in Django 3.0 so it will cause ImportError

## Solution

Drop six and add django 3.0 to the test matrix to ensure it is supported

## Commandments

- [x] Write PEP8 compliant code.
- [x] Cover it with tests.
- [x] Update `CHANGES.rst` file to describe the changes, and quote according issue with `GH-<issue_number>`.
- [x] Pay attention to backward compatibility, or if it breaks it, explain why.
- [x] Update documentation (if relevant).

### CODE REVIEW

**Comment by auvipy** [comment]
one last failure self = <django.db.models.query_utils.DeferredAttribute object at 0x7f6db0f84e80>

instance = <ModelWithCustomDescriptor: ModelWithCustomDescriptor object (2)>

    def _check_parent_chain(self, instance):

        """

        Check if the field value can be fetched from a parent field already

        loaded in the instance. This can be done if the to-be fetched

        field is a primary key field.

        """

        opts = instance._meta

>       link_field = opts.get_ancestor_link(self.field.model)

E       AttributeError: 'StringyDescriptor' object has no attribute 'model'

.tox/py37-django30/lib/python3.7/site-packages/django/db/models/query_utils.py:148: AttributeError

**Comment by auvipy** [comment]
can you plz resolve the conflicts?

**Comment by Surgo** [comment]
@mvbrn Any update?

**Comment by tony** [comment]
@mvbrn Hm, conflicts need to be resolved on this, and I actually just bumped into django-model-utils when checking into Django 3.0 (it's released now)

Any status report on if you can get around to this?

In the event you are too busy, is it okay if other contributors fork this PR or work off it?

**Comment by Natim** [comment]
`E AttributeError: 'StringyDescriptor' object has no attribute 'model'` with Python 3.8

**Comment by Natim** [comment]
Ok something is probably wrong with this StringlyDescriptor mock.

**Comment by Natim** [comment]
@lucaswiman since you wrote the test, do you have any insights about how to fix that?

**Comment by auvipy** [comment]
can not load travis

**Comment by Natim** [comment]
I am so sorry for the noise, since it is @mvbrn master branch I cannot push-force in it so I cannot merge my commit or squash anything...

**Comment by auvipy** [comment]
OK 4 test failing

**Comment by Natim** [comment]
Apparently the code coverage report decreasing is because we removed lines not test coverage.

**Comment by Natim** [comment]
> OK 4 test failing

They are now fixed.

File: tests/models.py [hunk 1]
```diff
@@ -373,26 +373,27 @@ class StringyDescriptor:
     """
     Descriptor that returns a string version of the underlying integer value.
     """
-
     def __init__(self, name):
-        self.name = name
```
Comment (Natim) [inline tests/models.py]: I would revert that change.

File: CHANGES.rst [hunk 2]
```diff
@@ -423,4 +426,4 @@ CHANGES
 0.3.0
 -----
 
-* Added ``QueryManager``
+- Added ``QueryManager``
```
Comment (Natim) [inline CHANGES.rst]: This is up to you, I can also use `* ` everywhere. Just a matter of taste but I feel that we shouldn't mix them.

**Comment by Natim** [comment]
Thank you @auvipy for the quick review!

**Pull request merged**